src/Fixer/Whitespace/MethodChainingIndentationFixer.php

```diff
@@ -187,11 +186,10 @@ private function getIndentContentAt(Tokens $tokens, $index)
     private function currentLineRequiresExtraIndentLevel(Tokens $tokens, $start, $end)
     {
         if ($tokens[$start + 1]->isGivenKind(T_OBJECT_OPERATOR)) {
-            return false;
-        }
-
-        if ($tokens[$end]->isGivenKind(CT::T_BRACE_CLASS_INSTANTIATION_CLOSE)) {
-            return true;
+            return
+                $tokens[$start + 3]->equals('(')
```

Comment:
This is definitely way too optimistic - you cannot assume that token `+3` is `(`, because before there can be many non-meaningful tokens - spaces and comments - e.g. someone might like to "wrap" `->` with spaces, so the `(` will not be on position `+3`, but `+4`/`+5`.

You need to get there via `Tokens::getNextMeaningfulToken`.

Same applies to `$tokens[$start + 1]` - let's add a test case with many whitespaces, and another one with many comments - similar what you can see when you search for "buahaha" string :)

Reply:
Is there a short way to get the 3rd next meaningful token? I can't find something simpler than just chaining 3 calls to `Tokens::getNextMeaningfulToken`.

Reply:
I'm afraid not.

Reply:
Added these with db57d54fe81d108d54f10c721ff410408dca0d74